server/container_attach.go

```diff
@@ -19,7 +18,6 @@ func (s *Server) Attach(ctx context.Context, req *pb.AttachRequest) (resp *pb.At
 		recordOperation(operation, time.Now())
 		recordError(operation, err)
 	}()
-	logrus.Debugf("AttachRequest %+v", req)
```

Comment:
was there a reason for dropping this?

Reply:
All requests are now automatically logged via the interceptor:
https://github.com/cri-o/cri-o/blob/53f82f7858c7bfa47fb98e0fbf7599500413907a/pkg/log/interceptors.go#L57-L65

This way we have a homogeneous look and feel for the standard log messages without doing any further work.